examples/bits.rs

```diff
@@ -1,98 +1,207 @@
+
+        // For each byte (`u8`), we use `bitvec` to make a view into its bits.
+        // `bitvec` provides the `.bits::<_>()` method on Rust integers for easy
+        // access to its view types.
+        //
+        // The `LittleEndian` means that the view moves from least significant
+        // bit to most significant. Since we want to display on screen the most
+        // significant bit on the left, and the least on the right, the turtle
+        // will have to move from right to left to match.
+        //
+        // These types describe different ways to view the same data. You can
+        // read more about them in the `bitvec` docs, and at Wikipedia:
+        // https://docs.rs/bitvec/0.16.1/bitvec/cursor/index.html
+        // https://en.wikipedia.org/wiki/Endianness#Bit_endianness
+        for byte in row {
+            println!("  Byte {:02}:\n    Value: 0x{:02X}\n    Bits: {:08b}", row_num, byte, byte);
+
+            let bits: &BitSlice<_, _> = byte.bits::<LittleEndian>();
 
-impl BitVec {
-    /// Create a new BitVec with the given size in bits
-    /// Note that the actual size of the vector is in bytes so `bits` will be rounded up to the
-    /// nearest multiple of 8 >= bits.
-    pub fn with_capacity(bits: usize) -> Self {
-        if bits > 0 {
-            // Ceiling division of bits / BITS so that we get >= bits capacity
-            // From: https://stackoverflow.com/a/2745086/551904
-            let capacity = 1 + ((bits - 1) / BITS);
-            Self { bytes: vec![0; capacity] }
-        } else {
-            Default::default()
+            // Then we draw the byte's bits as a row
+            draw_row(&mut turtle, bits);
+            // And reset the turtle to the right edge.
+            turtle.set_x(right_edge);
+
+            row_num += 1;
         }
     }
 
-    /// Returns the size in bits of the array
-    pub fn len(&self) -> usize {
-        self.len_bytes() * BITS
-    }
+    // `bitvec` can look at more than just `u8`. Let's try looking at the bits
+    // that represent a number!
+    //
+    // But `bitvec` doesn't know how to view all numbers. The standard library
+    // provides a function `to_bits(f64) -> u64`, which turns it into a number
+    // `bitvec` does know how to view.
+    //
+    // The `f64` data type has a memory encoding standardized by the IEEE-754
+    // document. You can read more about it here:
+    // https://en.wikipedia.org/wiki/Double-precision_floating-point_format
+    let raw_number: u64 = NUMBER.to_bits();
+
+    // `bitvec` can also view bits from left to right, with `BigEndian`.
+    let bits: &BitSlice<_, _> = raw_number.bits::<BigEndian>();
 
-    /// Returns the size in bytes of the array
-    pub fn len_bytes(&self) -> usize {
-        self.bytes.len()
+    // Since we are reading bits from left to right, the turtle should move from
+    // left to right also. Change the `* 2.0` to move the section horizontally.
```

Comment:
nit: "Change the" -> "You can change the" to make it clear that we're providing an instruction, not further explaining what this code does.